petdeface/mideface.py

```diff
@@ -170,9 +168,17 @@ def _list_outputs(self):
                 fname = self._filename_from_source(name)
                 if isdefined(fname):
                     outputs[out_name] = os.path.abspath(fname)
-            if self.inputs.pics is True and self.inputs.odir is not None and self.inputs.code is not None:
-                outputs["out_before_pic"] = os.path.abspath(f"{self.inputs.odir}/{self.inputs.code}.face-before.png")
-                outputs["out_after_pic"] = os.path.abspath(f"{self.inputs.odir}/{self.inputs.code}.face-after.png")
+            if (
+                self.inputs.pics is True
+                and self.inputs.odir is not None
```

Comment:
Good to see you expanding your expertise in nipype interfaces/workflows :-)

Reply:
Yeah, it's good to be on the other side of the learning curve.